_data/companies.yml

```
@@ -908,6 +908,14 @@
   events: Restricted
   last_update: 2020-03-11
 
+
+- name: OneFit
```

```suggestion
- name: OneFit
```

Dropping the extra blank line